pyscan/main.py

```diff
@@ -42,6 +46,12 @@ def print_debug(msg):
     if DEBUG:
         print(msg)
 
+def send_sensor_data(name, timeout):
+    while(pybytes):
```

Comment:
@Aditya18Jain I think this line is not needed either, since it's not using Pybytes. Sorry, I missed this before